authui/src/intlTelInput.ts

```diff
@@ -0,0 +1,28 @@
+export function setupIntlTelInput() {
+  const onlyCountries = JSON.parse(document.querySelector("meta[name=x-intl-tel-input-only-countries]")?.getAttribute("content") ?? "null") ?? [];
+  const preferredCountries = JSON.parse(document.querySelector("meta[name=x-intl-tel-input-preferred-countries]")?.getAttribute("content") ?? "null") ?? [];
+
+  let initialCountry = document.querySelector("meta[name=x-geoip-country-code]")?.getAttribute("content") ?? "";
+  // The detected country is not allowed.
+  if (onlyCountries.indexOf(initialCountry) < 0) {
+    initialCountry = "";
+  }
+
+  const elements = document.querySelectorAll("[data-intl-tel-input]");
+  for (let i = 0; i < elements.length; i++) {
+    const element = elements[i];
+    if (element instanceof HTMLInputElement) {
+      const name = element.name;
+      element.name = name + "_intl_tel_input";
+      const customContainer = element.getAttribute("data-intl-tel-input-class") ?? undefined;
+      window.intlTelInput(element, {
```

Comment:
I found that the phone input will be initialised twice when back from another page. We may need some logic like:
1. Reset the input when leaving the page
or
1. Mark something in the element to prevent initialising twice

Looks like 2 is easier.

To test, forgot password with the phone number. After submitting the form successful, click back.

<img width="472" alt="Screenshot 2021-07-20 at 11 50 45 AM" src="https://user-images.githubusercontent.com/388892/126250327-f6c7122d-ec23-4e4d-81d7-0869ed932458.png">